### PR TITLE
api: add hardcoded versioning support

### PR DESCRIPTION
**Pull request by GRISHNOV**
Add versioning support, now the module api has _VERSION hardcoded variable

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+* Add hardcoded versioning support, now the exported table
```
Comment (DifferentialOrange) [inline CHANGELOG.md:9]: I don't think we should tell user about implementation ("hardcoding" part). And "exported table" is also confusing, it's the module api.

File: crud.lua [hunk 2]
```diff
@@ -23,6 +23,8 @@ local stats = require('crud.stats')
 
 local crud = {}
 
+crud['_VERSION'] = require('crud.version')
```
Comment (DifferentialOrange) [inline crud.lua:26]: Let's use the same dotted notation as everything below and add a simple luadoc description like in others.

**Review by DifferentialOrange — COMMENTED** [review]
Let's finish with queue first.

File: CHANGELOG.md (hunk 1 above)
Comment (GRISHNOV) [reply CHANGELOG.md]: Simplified, also updated PR description

File: crud.lua (hunk 2 above)
Comment (GRISHNOV) [reply crud.lua]: Done

File: crud.lua [hunk 3]
```diff
@@ -23,6 +23,11 @@ local stats = require('crud.stats')
 
 local crud = {}
 
+-- @refer crud.version
+-- @tfield string _VERSION
+--  Contains hardcoded value of the CRUD version.
```
Comment (DifferentialOrange) [inline crud.lua:28]: Let's now write about implementation here

File: .github/workflows/check_on_push.yaml [hunk 4]
```diff
@@ -5,6 +5,17 @@ on:
   pull_request:
 
 jobs:
+  version-check:
```
Comment (DifferentialOrange) [inline .github/workflows/check_on_push.yaml:8]: I think we may ignore adding the job if nothing depends on it

Comment (GRISHNOV) [reply .github/workflows/check_on_push.yaml]: Deleted job from `check_on_push`

File: crud.lua (hunk 3 above)
Comment (GRISHNOV) [reply crud.lua:28]: Updated

Comment (DifferentialOrange) [reply crud.lua:28]: Well, sorry, it was a typo. I meant to write "let's not write". Let's not write about "hardcoded" in any public API description. And, well, I think it's better to remove the details about its maintenance too. Sorry

Comment (DifferentialOrange) [reply crud.lua:28]: Let's use just `module version`

File: CHANGELOG.md [hunk 5]
```diff
@@ -6,6 +6,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+* Add versioning support (PR #342).
```
Comment (DifferentialOrange) [inline CHANGELOG.md:9]: Add `### Added` section

File: crud.lua (hunk 3 above)
Comment (GRISHNOV) [reply crud.lua]: Done

File: CHANGELOG.md (hunk 5 above)
Comment (GRISHNOV) [reply CHANGELOG.md:11]: Done

File: .github/workflows/push_rockspec.yaml [hunk 6]
```diff
@@ -11,6 +11,16 @@ env:
   ROCK_NAME: "crud"
 
 jobs:
+  version-check:
+    # We need this job to run only on push with tag.
+    if: ${{ github.event_name == 'push' && startsWith(github.ref, 'refs/tags/') }}
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Check module version
+        uses: tarantool/actions/check-module-version@igrishnov/gh-204-add-hardcoded-versioning-support
```
Comment (DifferentialOrange) [inline .github/workflows/push_rockspec.yaml:20]: Let's update to master and merge

Comment (GRISHNOV) [reply .github/workflows/push_rockspec.yaml]: Done

**Pull request merged**